manifest.json

```diff
@@ -0,0 +1,21 @@
+{
+  "manifest_version": 2,
+  "name": "Regretful comment blocker",
```

Comment:
Extensions name TBD: https://github.com/dwyl/comments-you-will-regret-blocker-chrome-extension/issues/7#issuecomment-268750075